Fixed Errors in README.md (#278)
OfficeDev/office-js-snippets: README.md

4. Install `yarn` as a global package `npm install yarn --global`.
5. Be sure your CLI is in the root of the office-js-snippets repo and run `yarn install`. (It is similar to `npm install`.)

> 6. Set up the original \OfficeDev\office-js-snippets as the upstream repo for your local repo by following the steps in [Configuring a remote for a fork](https://help.github.com/articles/configuring-a-remote-for-a-fork/).

Reviewer: Current Microsoft style is to use contractions. It makes the text seem less formal and friendlier.

7. If you will be using Visual Studio Code as your editor, install the [TSLint](https://marketplace.visualstudio.com/items?itemName=eg2.tslint) extension for Visual Studio Code.

### Adding a new sample

> 

Reviewer: Including the "that" at the beginning of a relative clause makes it easier for readers for whom English is a second language.

6. Save the file **somewhere outside of the office-js-snippets project**. (You will move it into the project in a later step.) The file name must have a ".yaml" extension and it must be in [`kebab-case`](http://wiki.c2.com/?KebabCase). For examples, see the existing *.yaml files in the subfolders of the `samples` folder of the project.
7. Make sure the master branch of your fork is in sync with the master branch of the upstream \OfficeDev\office-js-snippets repo by following the steps in [Syncing a fork](https://help.github.com/articles/syncing-a-fork/).
8. Create a new branch at the **office-js-snippets** root folder of your local repo by running the command `git checkout -b {name_of_your_new_branch}`. (This will create and checkout the new branch. *Stay in this branch for all the remaining steps.*) Each snippet should have its own branch. Suggestion: use the name of the yaml file that you created above (without the extension) as the branch name.

> 9. Decide the project folder to store your snippet. All snippets must be inside the `samples` folder. The structure of the subfolders is:

Reviewer: ```suggestion
9. Decide the project folder where your snippet should be stored. All snippets must be inside the `samples` folder. The structure of the subfolders is:
```

 - The base folders such as `Excel`, `Word` etc. are all the various broad-level categories.

>  - Group folders for the group containing a snippet are inside each base folder.

Reviewer: ```suggestion
 - Inside of each base folder there are group folders.
```

>  - '.yaml' representing a snippet are inside each group folder.

Reviewer: Inside each group folder are '.yaml' files. Each one represents a snippet.
